.travis.yml

```diff
@@ -64,6 +62,9 @@ jobs:
 
     - name: Coverage
       rust: nightly
+      install:
+        - nvm install 10 && nvm alias default 10 && npm install -g yarn@latest && yarn --version
+        - (cd dex-contracts && yarn --frozen-lockfile && yarn run prepack)
```

Comment:
nit: Can this be part of the upper level `install` and then use `{before,after}_install` for the "Build and Local Testing" job specific steps (like installing clippy and rustfmt as well as doing the aws login)?